ThingIFSDK/ThingIFSDK/TimeRange.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TimeRange.swift
+//  ThingIFSDK
+//
+//  Created on 2017/01/30.
+//  Copyright (c) 2017 Kii. All rights reserved.
+//
+
+import Foundation
+
+open class TimeRange: NSCoding {
+
+    /** Start point of a time range. */
```

Comment:
If we would like to provide more use cases, we need to provide these options for developer. Such as adding attribute to indicate inclusive or exclusive, left open or write open. 

But in our API, the timeRange is just used as inclusive case. We can provide more options, if we allow our developers for other options after server provide these possibilities.